tun/tests/configure.rs

```diff
@@ -9,6 +9,19 @@ fn test_create() {
     TunInterface::new()?;
 }
 
+#[test]
+#[throws]
+#[cfg(not(any(target_os = "windows", target_vendor = "apple")))]
+fn test_set_get_broadcast_addr() {
+    let tun = TunInterface::new()?;
+
+    let addr = Ipv4Addr::new(0, 0, 0, 0);
```

Comment:
```suggestion
    let addr = Ipv4Addr::new(0, 0, 0, 255);
```

ChatGPT suggests that these addresses need to end in 255... maybe?

Reply:
@sampoder try `255.255.255.0`

Reply:
```suggestion
    let addr = Ipv4Addr::new(255, 255, 255, 0);
```

Reply:
bummer! no luck